libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Base.hs

```diff
@@ -349,6 +347,18 @@ ratifyStateSpec _ RatifyEnv {..} =
     disableBootstrap pp = match pp $ \pp' ->
       match (sel @12 pp') $ \major _ ->
         assert $ not_ (major ==. lit (natVersion @9))
```

Comment:
I think below 9 doesn't make sense either, no?  For most tests I set it to 10 in the environment spec,, via `satisfies pp pparamsSpec`, (along with some other constraints for pparams) -  maybe it would make sense to have this here as well?